### PR TITLE
Add uninstallation instructions. Fix #38.

### LINKED ISSUE #38: Add uninstallation instructions for "batteries included"
The templates comes with `Web`, `Wrappers-JQuery` (and jQuery itself), `IDE`, `Benchmark` and `Examples` included. If #35 is included, it can include Silk as well.

All of those are in for the initial project to allow some preexisting code and possibilities, but in the end some of them should be ultimately uninstalled after the author chooses what to pick.

There should be uninstallation instructions included somewhere.

### PR DESCRIPTION
**Pull request by herby**

### CODE REVIEW

**Comment by herby** [comment]
Ping @mkroehnert @hhzl.

File: root/parts.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<html>
+
+  <head>
+    <title>{%= title %}</title>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta name="author" content="{%= author_name %}" />
+  </head>
+
+  <body>
+  <p>The sample app contains there additional parts beyond Amber itself:</p>
```
Comment (hhzl) [inline root/parts.html]: @herby I assume you mean *three* and not *there*.

File: root/parts.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <ul>
+      <li>The elements in <code>index.html</code>
+          <code>&lt;body&gt;</code> tag.</li>
+      <li><code>amber-contrib-jquery</code> module,
+      providing jQuery and wrapper over it.</li>
```
Comment (hhzl) [inline root/parts.html]: ... and **a** wrapper ....

File: root/uninstall.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          package from <code>amber-contrib-web</code> module,
+          but want to retain the jQuery wrapper,
+          you should run <code>bower uninstall amber-contrib-web --save</code>
+          from your CLI.</li>
+      <li>If you aren't going to use <code>Web</code>
+          package from <code>amber-contrib-web</code> module,
+          nor the jQuery wrapper from <code>amber-contrib-jquery</code>,
+          you should run <code>bower uninstall amber-contrib-web amber-contrib-jquery --save</code>
+          from your CLI.</li>
+      <li>You cannot remove just the jQuery wrapper and retain Web package,
```
Comment (hhzl) [inline root/uninstall.html]: ...retain **the** Web package

File: root/uninstall.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      as Web package depends on the jQuery wrapper.</li>
```
Comment (hhzl) [inline root/uninstall.html]: as **it** depends on

File: root/uninstall.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      <li>As last step, make sure dependency tree is in correct state
```
Comment (hhzl) [inline root/uninstall.html]: as **the** last step

**Comment by herby** [comment]
@hhzl fixed

File: root/uninstall.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+          you should run <code>bower uninstall amber-contrib-web --save</code>
+          from your CLI.</li>
+      <li>If you aren't going to use <code>Web</code>
+          package from <code>amber-contrib-web</code> module,
+          nor the jQuery wrapper from <code>amber-contrib-jquery</code>,
+          you should run <code>bower uninstall amber-contrib-web amber-contrib-jquery --save</code>
+          from your CLI.</li>
+      <li>You cannot remove just the jQuery wrapper and retain the Web package,
+      as it depends on the jQuery wrapper.</li>
+      <li>As the last step, make sure dependency tree is in correct state
```
Comment (hhzl) [inline root/uninstall.html]: **the** dependency tree  //  in **a** correct state.

File: root/uninstall.html [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+      remove all unnecessary parts left from the sample app.</p>
+  <ul style="width: 40ex">
+      <li>In <code>index.html</code>
+          <code>&lt;body&gt;</code> tag,
+          everything should be removed, except
+          the <code>&lt;script&gt;</code> tag
+          that initializes Amber and starts your app;
+          then you can add elements specific to your own app
+          to the page.</li>
+      <li>If you aren't going to use <code>Web</code>
```
Comment (hhzl) [inline root/uninstall.html]: **the** Web

File: root/uninstall.html [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+          package from <code>amber-contrib-web</code> module,
```
Comment (hhzl) [inline root/uninstall.html]: from **the**

File: root/uninstall.html [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+          but want to retain the jQuery wrapper,
+          you should run <code>bower uninstall amber-contrib-web --save</code>
+          from your CLI.</li>
+      <li>If you aren't going to use <code>Web</code>
+          package from <code>amber-contrib-web</code> module,
```
Comment (hhzl) [inline root/uninstall.html]: frim **the**

File: root/uninstall.html [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+          nor the jQuery wrapper from <code>amber-contrib-jquery</code>,
+          you should run <code>bower uninstall amber-contrib-web amber-contrib-jquery --save</code>
+          from your CLI.</li>
+      <li>You cannot remove just the jQuery wrapper and retain the Web package,
+      as it depends on the jQuery wrapper.</li>
+      <li>As the last step, make sure dependency tree is in correct state
+          by regetting it: first delete your <code>bower_components</code> folder,
```
Comment (hhzl) [inline root/uninstall.html]: regetting --> recreating

**Pull request merged**